sdk/farmbeats/azure-farmbeats/azure/farmbeats/apigroupclients/boundaryclient.py

```diff
@@ -0,0 +1,72 @@
+class BoundaryClient():
+    def __init__(
+        self,
+        farmbeats_client
+    ):
+        self.client = farmbeats_client
+
+    def create(
+        self,
+        farmer_id,
+        boundary_id,
+        boundary
+    ):
+        return self.client.create_boundary(
+            farmer_id,
+            boundary_id,
+            boundary,
+        )
+
+    def get_all(
```

Comment:
This methods precludes the ability to support server driven paging. We return `ItemPaged` instances explicitly to not paint ourselves into that corner.